cmd/portal/internal.go

```diff
@@ -51,4 +80,13 @@ func init() {
 
 	_ = cmdInternalSetupPortal.MarkFlagRequired("default-authgear-domain")
 	_ = cmdInternalSetupPortal.MarkFlagRequired("custom-authgear-domain")
+
+	cmdInternalBreakingChangeMigrateK8SToDB.Flags().StringVar(&DatabaseURL, "database-url", "", "Database URL")
+	cmdInternalBreakingChangeMigrateK8SToDB.Flags().StringVar(&DatabaseSchema, "database-schema", "", "Database schema name")
+	cmdInternalBreakingChangeMigrateK8SToDB.Flags().StringVar(&kubeConfigPath, "kubeconfig", "", "Path to kubeconfig")
+	cmdInternalBreakingChangeMigrateK8SToDB.Flags().StringVar(&namespace, "namespace", "", "Namespace")
+
+	// FIXME: Respect KUBECONFIG environment variable.
+	_ = cmdInternalBreakingChangeMigrateK8SToDB.MarkFlagRequired("kubeconfig")
```

Comment:
I think `kubeconfig` should not be required? As `MakeKubeConfig` accept empty `kubeConfigPath` and home kube config (e.g. `~/.kube/config`) will be returned.

Reply:
But it still do not respect the `KUBECONFIG` environment variable. I googled and found out that people typically use cobra with viper to accept both environment variable and command line argument. I think we should update our code to that. So I added a FIXME here.